controllers/front/OrderOpcController.php

```diff
@@ -95,10 +95,21 @@ private function checkWhetherAllConditionsAreApproved()
         return true;
     }
 
+    public function getPaymentOptionsAction()
+    {
+        return $this->renderPaymentOptions();
+    }
+
     public function init()
     {
         parent::init();
 
+        if (($action = Tools::getValue('action'))) {
+            $result = $this->{$action . 'Action'}();
```

Comment:
Yes PrestaShop has a tendency to silence all errors. It's OK when the error can be caused by a user, but users have no reason to mess with URL parameters I think.